From 6996d06d91b87b441834ca9c37fcb7db1fa800c8 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 13 Aug 2015 17:57:03 +0800
Subject: [PATCH 0179/5242] MLK-11349-4 ARM: imx: need to make sure GPT1 clk
 root enabled

commit  ebd4afaa34e11e821489c0b559d6557fb56bbc93 from
https://source.codeaurora.org/external/imx/linux-imx.git

GPT1 clk root must be enabled, as GPT1 is system's clk
source.

Signed-off-by: Anson Huang <b20788@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7s.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7s.dtsi b/arch/arm/boot/dts/imx7s.dtsi
index 8ab641e..d0844b7 100644
--- a/arch/arm/boot/dts/imx7s.dtsi
+++ b/arch/arm/boot/dts/imx7s.dtsi
@@ -444,7 +444,7 @@
 				compatible = "fsl,imx7d-gpt", "fsl,imx6sx-gpt";
 				reg = <0x302d0000 0x10000>;
 				interrupts = <GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
+				clocks = <&clks IMX7D_GPT1_ROOT_CLK>,
 					 <&clks IMX7D_GPT1_ROOT_CLK>;
 				clock-names = "ipg", "per";
 			};
-- 
1.7.9.5

